src/aind_data_schema_models/modalities.py

```diff
@@ -25,3 +27,66 @@ class ModalityModel(BaseName):
 
 Modality.abbreviation_map = {m().abbreviation: m() for m in Modality.ALL}
 Modality.from_abbreviation = lambda x: Modality.abbreviation_map.get(x)
+
+
+class FileRequirement(IntEnum):
+    """Whether a file is required for a specific modality"""
+
+    REQUIRED = 1
+    OPTIONAL = 0
+    IGNORED = -1
```

Comment:
I think we want a differently label. Ignored sounds equivalent to optional to me - e.g. doesn't matter if it's there or not - whereas what we're saying is files labeled with -1 should actively not be present. 
Not sure the right word though. Barred?

Reply:
How about "Not Applicable" or "Excluded"?